rskj-core/src/main/java/org/ethereum/rpc/TypeConverter.java

```diff
@@ -82,11 +108,11 @@ public static String toJsonHex(byte[] x) {
     }
 
     public static String toJsonHex(Coin x) {
-        return x != null ? x.asBigInteger().toString() : "" ;
+        return x != null ? x.asBigInteger().toString() : "";
     }
 
     public static String toJsonHex(String x) {
-        return "0x"+x;
+        return "0x" + x;
```

Comment:
Maybe add a sanity check here that `x` is actually a number in hex? `Character.digit` could be useful. Or convert string to big integer and just call `toQuantityJsonHex(BigInteger n)`.

Reply:
Updated